packages/@aws-cdk/tmp-toolkit-helpers/src/api/hotswap/ecs-services.ts

```diff
@@ -153,7 +153,7 @@ export async function isHotswappableEcsServiceChange(
             await sdk.ecs().waitUntilServicesStable({
               cluster: update.service?.clusterArn,
               services: [service.serviceArn],
-            });
+            }, timeoutSeconds);
```

Comment:
Can we also bring down the default to something reasonable? Like 2 minutes?

Reply:
I would like to, but lets be careful about this because its a breaking change. History tells us that a small section of customers will indeed be broken. 

- **On one hand**, the folks benefiting from a reduced timeout are those who's operation fails right now anyway, so presumably they have fixed whatever the problem was and it doesn't hurt them that much.
- **On the other hand**, users who's update takes >2 minutes will now start seeing failures.

All in all: broken operations continue to be broken, and working operations may fail. Doesn't seem worth it...

@mrgrain wanna be the tie breaker?

Reply:
I only realize this now: The way we are using this value isn't really a generic hotswap timeout, isn't it? It's a very specific option on how long to wait for ECS service stabilization - which can be very different depending on your actual service.  

Tend to agree with @iliapolo that this isn't worth the risk of breaking.